packages/calcite-components/src/components/list/list.tsx

```diff
@@ -731,15 +727,23 @@ export class List
     this.updateFilteredData();
   }
 
+  private get effectiveFilterProps(): string[] {
+    if (!this.filterProps) {
+      return ["description", "label", "metadata"];
+    }
+
+    return this.filterProps.filter((prop) => prop !== "el");
```

Comment:
Not sure I follow. Could you expand on this?

Reply:
Gotcha, thanks for clarifying. I think we would end up going the extracted-util route regardless since not all components use the `filter` component internally (e.g., `combobox`). I could update `filter` in #9830 to use the proposed util and interfaces, so these are all consistent.